### DISCUSSION TITLE
Programming Club Discussion Website for College Students

### DISCUSSION THREAD

**Discussion by kumarswaraj16** [scaleracademy/open-source-projects#63] [Ideas]
**The idea is to create a platform for college students so that they can interact with each other while they are facing problems while implementing their codes in Data Structure and Algorithms and Competitive Programming. Students can also share any problems related to any development like Web development , App Development , Machine Learning.
Teachers, students and College authorities can also share any information related to College information.
Alumni can also share any information and Placed students can also share their interview.
The idea is to bring a healthy environment of programming among college students in all possible way.**

**Features**

- Register/SignUp
- Login 
- Log out
- Post 
- Like a post
- Delete your Post
- Search any Post
- Update Your Profile
- Know about Author of any Post/Blog
- Authentic Person will have a green tick after their name
- Comment below any Post/Blog
- Reply to any Comment
- Upvote and Downvote any Comment
- See Post according to categories
- See trending Posts

**Comment by aatransh** [comment]
Nice one..... Would be beneficial for The students...

**Comment by ItsmeCaptain1** [comment]
Nice Idea man !!

**Comment by ajaygour18** [comment]
good idea !

**Comment by princee-kabra** [comment]
Good one

**Comment by jagrti-23** [comment]
good idea!

**Comment by parikshit14** [comment]
nice idea @kumarswaraj16

**Comment by srishti397** [comment]
Great work dude👍

**Comment by Jaisinghal13** [comment]
Superb amazing idea swaraj. This must be helpful for students.

**Comment by narpatpatel03191** [comment]
Great idea  👍

**Comment by RakshitaRathore** [comment]
Good one bro 👍

**Comment by priyansh157** [comment]
nice idea bro....👍👍

**Comment by manojyadav12** [comment]
Nice idea .

**Comment by hinkal338** [comment]
Nice idea ..!!

**Comment by mohitvats2102** [comment]
It will be helpful for upcoming students!!

**Comment by sadawat2709** [comment]
Great idea 👍👍

**Comment by nikitash99** [comment]
It's really Amazing!! Swaraj

**Comment by il3ven** [comment]
I think a community like you proposed can be easily built on Discord.

**Reply by christusdsouza** [reply]
and many more, one of them is reddit!

**Reply by il3ven** [reply]
Heck even a repo with discussions enabled can also be used. I don't see the need to build this except for practice.

**Comment by sev7enwonder** [comment]
https://thejoboverflow.com we can take motivation from this site

**Comment by MrKrishnaAgarwal** [comment]
Good Idea!
We can implement it 👍🏻

**Comment by techysubham** [comment]
Great One!
Lets work on this Idea...
Cheers..👍🏻🥳

**Comment by iamrahul8** [comment]
Great Idea 💡
Full support ✌

**Comment by cy2000yadav** [comment]
Definitely a good idea!!

**Comment by sangita9853** [comment]
That's nice , For sure a good idea!!

**Comment by kaivalya98** [comment]
A good one , I liked the idea!

**Comment by saraswati258** [comment]
Nice and a great idea to look forward!

**Comment by manmohan4142** [comment]
Nice idea and good for all students.

**Comment by AdityaMathur25** [comment]
Looks really intresting .

**Comment by kishanrajput23** [comment]
Awesome idea brother 👏

**Comment by PrakashGatiyala** [comment]
We can have voice comment feature below any Post/Blog.

Good Idea Man.

**Comment by Raj-saurabh** [comment]
great idea will be much benificial for tier-3 collage students if got scaled accordingly!

**Comment by Rimjhim20** [comment]
great idea ! It Would be beneficial and create a good coding environment for students.

**Comment by Rakesh9100** [comment]
Great! It will be useful for every student.

**Comment by trishit78** [comment]
It's a great idea, but I think instead of teachers if we add tech professionals, it will be much better. So that we will be having good connections with them and ask for referrals and other important stuff(related to interviews and all).

**Reply by Zeeshan-Anwer** [reply]
But what we offer to tech professionals?? Student will be interested to join but I am not sure about tech professionals

**Comment by anisnorn** [comment]
Great idea, I also developed 1st time website of [mbwhatsking](https://mbwhatsking.com/id/) I know how interesting and amazing it's to work on 1st project
